### ISSUE TITLE
duplicate symbol _messageForError

### ISSUE THREAD

**Issue by codehob** [mCodex/react-native-sensitive-info#6]
duplicate symbol _messageForError in:
    /Users/iphonedeploy/Library/Developer/Xcode/DerivedData/punchinout-dbqwmosndkfyrhcrkbcebsnepfyv/Build/Products/Debug-iphonesimulator/libRNKeychain.a(RNKeychainManager.o)
    /Users/iphonedeploy/Library/Developer/Xcode/DerivedData/punchinout-dbqwmosndkfyrhcrkbcebsnepfyv/Build/Products/Debug-iphonesimulator/libRNSensitiveInfo.a(RNSensitiveInfo.o)
duplicate symbol _makeError in:
    /Users/iphonedeploy/Library/Developer/Xcode/DerivedData/punchinout-dbqwmosndkfyrhcrkbcebsnepfyv/Build/Products/Debug-iphonesimulator/libRNKeychain.a(RNKeychainManager.o)
    /Users/iphonedeploy/Library/Developer/Xcode/DerivedData/punchinout-dbqwmosndkfyrhcrkbcebsnepfyv/Build/Products/Debug-iphonesimulator/libRNSensitiveInfo.a(RNSensitiveInfo.o)
ld: 2 duplicate symbols for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by zibs** [comment]
I'm also getting this - is there a way to resolve? Why was this closed exactly?

**Comment by mCodex** [comment]
Hello @zibs , I couldn't reproduce that. Could you provide an example?

**Comment by zibs** [comment]
@mCodex Sorry it took me so long to get back to you:
https://github.com/zibs/keychain-and-info-example

You should be able to just clone + `yarn` + run this on your iPhone and you will see the error. It will work on the simulator, and it works on the Android device as well. But it does not work on an actual iPhone. 

Thanks!

**Comment by mCodex** [comment]
@zibs thank you!

**Comment by DavidPotter** [comment]
My team is running into this as well.  It appears that this is fixed in the repo but a new version has not been published.  When will you be publishing a new version with all the changes from the past 4+ months?

**Comment by DavidPotter** [comment]
Hmm, I may have posted my comment on the wrong issue.  I'm getting a different warning.  I'll open a new issue for that.  Sorry.

**Comment by mCodex** [comment]
Hello Guys, is this still happening in v5.2.0 (currently beta)?

**Comment by scstan** [comment]
Happens for me also on
"react-native": "0.49.3",
"react-native-sensitive-info": "^5.1.0"

**Comment by scstan** [comment]
5.2.0 also

**Comment by crisppY** [comment]
Yes. I'm having build errors in v5.2.0.

```
ld: warning: object file (~/app/ios/build/Build/Products/Debug-iphonesimulator/libRNKeychain.a(RNKeychainManager.o)) was built for newer iOS version (8.3) than being linked (8.0)
duplicate symbol _messageForError in:
    ~/app/ios/build/Build/Products/Debug-iphonesimulator/libRNKeychain.a(RNKeychainManager.o)
    ~/app/ios/build/Build/Products/Debug-iphonesimulator/libRNSensitiveInfo.a(RNSensitiveInfo.o)
ld: 1 duplicate symbol for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by enricotelesca-ada** [comment]
same here, did anyone solve?

**Comment by alexmbp** [comment]
Have the same issue. Together with react-native-keychain. I believe - issue is with cocoapods while RNKeychain using pods and RN Sensitive Info does not. We have to add local pod spec to make it working. Not sure about it

**Comment by lucasbento** [comment]
I had the same issue, this indeed conflicts with [react-native-keychain](https://github.com/oblador/react-native-keychain) (thanks @alexmbp).

**Comment by mCodex** [comment]
Can anyone make a PR for that? Please :D

**Comment by mCodex** [comment]
This fix is available on latest beta version (5.2.5). 

```
yarn add react-native-sensitive-info@beta
```

Tks @grigorigoldman
